docs/components/Countdown/Countdown.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Utilities/Countdown/Docs" />
+
+# Countdown
+
+Countdown is used to ensure that a "countdown" to a specified end time is
+displayed in a consistent format. No text styling is applied to Countdown; it is
+only concerned with the text content that will be presented.
+
+## Design & usage guidelines
+
+Use Countdown when:
+
+- There is a need to make a deadline, such as time left to update important
```

Comment:
Periods on the bullet points 😆 Probably on the points in Related components too.

```suggestion
- There is a need to make a deadline, such as time left to update important.
```